inst/scripts/clusterSystemsScript.R

```diff
@@ -0,0 +1,87 @@
+# clusterSystemsScript.R
```

Comment:
You can remove the "template" code that is not actually necessary for your script itself.